wallet/lib/pages/detailed_asset_view/widgets/nft_3d_asset.dart

```diff
@@ -63,20 +63,26 @@ class Nft3DWidgetContent extends StatelessWidget {
           relatedJs: "",
           onProgress: (status) {
             if (status == 1) {
-              viewModel.toggleLoader();
+              Future.delayed(const Duration(seconds: 1),(){
```

Comment:
What about using schedule microtask?

Reply:
@kjawadDeveloper2 I tried using scheduleMicroTask() as it is in executes in MicroTask Queue it toggle our variable while rendering 3D model that produce a little Jank in our follow so we have to wait for almost 1 to toggling a variable for create a smooth flow.